modules/hal_infineon/Kconfig

```diff
@@ -4,6 +4,9 @@
 config ZEPHYR_HAL_INFINEON_MODULE
 	bool
 
+config ZEPHYR_HAL_INFINEON_MODULE_BLOBS
+	bool
```

Comment:
This will require some documentation in https://docs.zephyrproject.org/latest/contribute/bin_blobs.html to explain when and why these symbols should be set.
For BLE/HCI, people will be able to reproduce the existing config, but for other areas this could likely be missed

Reply:
> This will require some documentation in https://docs.zephyrproject.org/latest/contribute/bin_blobs.html to explain when and why these symbols should be set. For BLE/HCI, people will be able to reproduce the existing config, but for other areas this could likely be missed

I agree that documentation is needed. That said, I haven't found any documentation anywhere regarding the need to separately define `ZEPHYR_<mod>_MODULE` either. I think  documentation for that as well as the new `_BLOBS` symbol is needed.

Btw, I removed all of these Kconfig file additions from this PR as we wait for @tejlmand to come back with some proper solution.

Reply:
Ok, documentation need is indeed wider than current PR.